src/Data/Fin/Properties.agda

```diff
@@ -443,6 +443,9 @@ toℕ-inject {i = suc i} (suc j) = cong suc (toℕ-inject j)
 -- inject₁
 ------------------------------------------------------------------------
 
+fromℕ≢inject₁ : (j : Fin n) → fromℕ n ≢ inject₁ j
```

Comment:
Can/should `j` be made implicit here?

Reply:
With the definition
```agda
fromℕ≢inject₁ : {j : Fin n} → fromℕ n ≢ inject₁ j
fromℕ≢inject₁ {j = suc j} eq = fromℕ≢inject₁ {j = j} (suc-injective eq)
```
being (less) preferable to read/write, but easier/more efficient to *use*?